### ISSUE TITLE
Changing to Fullscreen

### ISSUE THREAD

**Issue by Eggysgames** [Eggysgames/BouncyRumble#1]
hey @Maitreyi1507 

Good to meet you mate. I see you made some changes to the renderer, was that to fix a bug for 2021 unity? 

Also I was told via email with Rumble they were needing fullscreen, did you make any changes to a index.html code to achieve that or was that done with some sort of settings in Unity? Wanted to learn from it thanks! :)

**Comment by Maitreyi1507** [comment]
Yes, that render change is done automatically by unity for 2021 unity version. I did not change anything to index.html.

**Comment by Maitreyi1507** [comment]
Only switch to webgl and nothing changed

**Comment by Eggysgames** [comment]
Hmm very odd. I wonder why my webgl isnt fullscreen. Maybe its the newer unity version has a setting.
